### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by dsambugaro**
Just a simple README.md (:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Next
+
+[![Vue.js](https://img.shields.io/badge/Vue.js-35495E?style=for-the-badge&logo=vuedotjs&logoColor=4FC08D)](https://vuejs.org/)
+[![firebase](	https://img.shields.io/badge/firebase-ffca28?style=for-the-badge&logo=firebase&logoColor=black)](https://firebase.google.com/)
+
+This repository contains a website that the GETPro group uses to define the `Next` studies' and presentations'  themes.
```
Comment (brnomendes) [inline README.md]:
```suggestion
This repository contains a website that the GETPro group uses to define the `Next` studies and presentations  themes.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## How to run it
+
+First of all, put your firebase credentials at `database.js`.
+
+As this project does not use any package manager, you are good to go! Just start an HTTP server at the root of this project and access it.
+
+If you have `python` a simple way to do that is running this command:
```
Comment (brnomendes) [inline README.md]:
```suggestion
If you have `python`, a simple way to do that is running this command:
```

**Review by brnomendes — APPROVED** [review]
🔝

**Review by thiagohoh — APPROVED** [review]

**Review by romavini — APPROVED** [review]
🍕

**Pull request merged**